src/skin/popup.html

```diff
@@ -121,6 +124,7 @@ <h1 id="share_title" class="i18n_share_title overlay_title"></h1>
     <button id="error" class="pbButton" target="blank"><span class="i18n_report_broken_site"></span></button>
     <button id="donate" class="pbButton" target="blank"><span class="i18n_donate_to_eff"></span></button>
 </div>
+<span id="version" class="i18n_version">
```

Comment:
Malformed HTML.

Reply:
There is now a vertical scrollbar for the entire popup in Firefox. The scrollbar is an existing issue (https://github.com/EFForg/privacybadger/issues/1077#issuecomment-340041201), but not in the default locale. Ideally, we would find a solution that works for all locales, but a quick tweaks that restores the default locale to present (no scrollbar) state is OK too. (Forcing the scrollbar to be hidden is not an acceptable tweak ... we don't want to have any cases where Badger buttons are entirely inaccessible.)

Reply:
Ah, I'm seeing the vertical scrollbar now. It seems to change with the new badger img. I'm tweaking some of the padding and margin values on the header and tracker portions of the popup to narrow it down to bare essentials without making it seem too cramped. There are some sizing issues with the newer badger icon (with the less beady eyes), once I get that sorted out with design I'll share screenshots here.

Reply:
Closing tag and scrollbar in Firefox are still problems.